src/base-component/AuthMediumLayout.jsx

```diff
@@ -2,68 +2,45 @@ import React from 'react';
 
 import { getConfig } from '@edx/frontend-platform';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
-import {
-  Col, Hyperlink, Image, Row,
-} from '@edx/paragon';
+import { Hyperlink, Image } from '@edx/paragon';
 import PropTypes from 'prop-types';
 
 import messages from './messages';
 
-const AuthMediumLayout = (props) => {
-  const { intl, username } = props;
-
-  return (
-    <div className="container row p-0 mb-3 medium-container">
-      <div className="col-md-10 p-0 screen-header-light">
+const AuthMediumLayout = ({ intl, username }) => (
+  <>
+    <div className="w-100 large-screen-top-stripe large-screen" />
```

Comment:
Is this large screen's stripe element intentional? If yes, I could not understand its use case here.

Reply:
It's the original classname. I will update it to `medium-screen-top-stripe`

Reply:
Simplified the classes.